app/components/settings/project_custom_fields/edit_form_header_component.html.erb

```diff
@@ -43,7 +43,7 @@ See COPYRIGHT and LICENSE files for more details.
         selected: project_custom_field_project_mappings_selected?,
         href: project_mappings_admin_settings_project_custom_field_path(@custom_field)
       ) do |tab|
-        tab.with_text { I18n.t("projects.settings.project_custom_fields.project_mappings.header") }
+        tab.with_text { t(:label_project_mappings) }
```

Comment:
I wasn't sure if it was too early to extract that to a common label, but it seems like this could be used at multiple locations in the future as well.

Reply:
I can totally relate to not wanting too many "label_" translations. Things like this can go wild. But I checked again and found not other place that seemed to fit. So I also think we should keep it there for now.